wiki/en/FAQ.md

```diff
@@ -28,9 +28,9 @@ Bear in mind also that all participants should follow [The Golden Rule](/wiki/Cl
 
 ### How do I know if I can join a server? Are there rules?
 
-In general, if somebody sets up a public server, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.
+In general, if somebody sets up a Registered Server, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.
 
-You can also set up a private server on Jamulus and give others your address to connect to that. Have a look at [this page for more information](/wiki/Running-a-Server).
+You can also set up Unregistered Server on Jamulus and give others your address to connect to that. Have a look at [this overview for more information](/wiki/Running-a-Server#server-types).
```

Comment:
> You can also set up Unregistered Server

I'd say something more like:

You do not have to Register your Server - you can give the address to others and they can connect directly.